src/config/modules/index.ts

```diff
@@ -24,6 +24,28 @@ export const modulesConfig: ReturnType<DefineNuxtConfig> = {
     classSuffix: '',
   },
   ...cookieControlConfig,
+  gtag: {
+    config: {
+      cookie_flags: `samesite=strict${
+        process.env.NODE_ENV === 'production' ? ';secure' : ''
```

Comment:
It might be possible and cool to set this at runtime, but it should work this way as well.

Reply:
It does not appear to me that this configuration could be set at runtime.